developers/weaviate/configuration/schema-configuration.md

```diff
@@ -234,6 +234,19 @@ This configuration allows stopwords to be configured by class. If not set, these
 - If the same item is included in both additions and removals, then an error is returned
 :::
 
+As of `v1.18`, stopwords are indexed, but are skipped in BM25. Meaning, stopwords are included in the inverted index, but when the BM25 algorithm is applied, they are not considered for relevance ranking. 
+
+Here is an example: 
+
+`query`: “Where is the nearest fast food chain?”
+
+`stopwords`: [“where”, “is”, “the”] 
+
+Is the same as: 
+
+`query`: “Nearest fast food chain”
+
+
```

Comment:
Is this meant to be used as part of a query?
If yes, then perhaps we should show a simplified
- query GraphQL alone should be fine
- we could just show the hybrid, or bm25 part of the query